packages/flame_forge2d/example/lib/widget_sample.dart

```diff
@@ -31,8 +31,9 @@ class WidgetSample extends Forge2DGame with TapDetector {
   Body createBody() {
     final bodyDef = BodyDef()
       ..angularVelocity = 3
-      ..position =
-          screenToWorld(Vector2.random()..multiply(viewport.effectiveSize))
+      ..position = screenToWorld(
+        Vector2.random()..multiply(camera.viewport.effectiveSize),
```

Comment:
I wonder if we should expose this property, that seems to be used everywhere, move readily inside baseGame. like canvasSize or originalSize, alongside the `size` (that is transformed)

Reply:
So far this has been a mechanical rename in order not to break any existing behavior. However, going forward it might be useful to review the notion of various sizes and `viewport.effectiveSize` in particular. I am not so sure that its current ubiquitous use is necessarily warranted.

For one, there is `windowSize = viewport.effectiveSize / camera.zoom`, which is the size of the game world currently visible through the camera. This could be useful, for example, if you want to avoid rendering objects that are out of the camera's view. 

Then, there's `viewportSize`, which is the size of the viewport in its own coordinate frame. For a top-level camera, this would be in pixels. This property can be useful, for example, for laying out various HUD elements that are attached to the viewport.

I'm not sure about the scenarios where the original `effectiveSize` would be useful.

In any case, I would prefer to leave the complexities of various size definitions and where they should be used outside the scope of this PR.